packages/web/components/pages/asset-info-page/chart.tsx

```diff
@@ -188,9 +188,7 @@ const ChartHeader = observer(() => {
       data = decHoverPrice;
     } else {
       if (assetInfoConfig.dataType === "price") {
-        data =
-          marketAsset?.currentPrice?.toDec() ??
-          new Dec(assetInfoConfig.lastChartData?.close ?? 0);
+        data = new Dec(assetInfoConfig.lastChartData?.close ?? 0);
```

Comment:
Have we verified whether this price reflects the most current value? I’m concerned that discrepancies might appear throughout the app

Reply:
There has been a discussion going on with Numia talking about that. As of now data from Numia and SQS differ so the last point on the chart and the price displayed when no hover is performed are different and inconsistent. They tried to make the SQS value the same as Numia but they ultimately decided to just use the Numia value in this page